src/keywrap/keyprovider.rs

```diff
@@ -135,6 +145,39 @@ impl KeyProviderKeyWrapProtocolOutput {
         serde_json::from_slice(&resp_bytes)
             .map_err(|_| anyhow!("keyprovider: failed to deserialize message from binary executor"))
     }
+
+    fn from_native(annotation: &str, dc_config: &DecryptConfig) -> Result<Self> {
```

Comment:
We may separate different keyproviders with feature config to avoid to build all of them, we can implement it in another PR.